crypto/secp256k1/curve.py

```diff
@@ -547,25 +567,28 @@ def addZ1EqualsZ2(self, x1, y1, z1, x2, y2, x3, y3, z3):
         fv = FieldVal
         a, b, c, d, e, f = fv(), fv(), fv(), fv(), fv(), fv()
         negX1, negY1, negE, negX3 = fv(), fv(), fv(), fv()
-        negX1.set(x1).negate(1)                # negX1 = -X1 (mag: 2)
-        negY1.set(y1).negate(1)                # negY1 = -Y1 (mag: 2)
-        a.set(negX1).add(x2)                  # A = X2-X1 (mag: 3)
-        b.squareVal(a)                        # B = A^2 (mag: 1)
-        c.set(negY1).add(y2)                  # C = Y2-Y1 (mag: 3)
-        d.squareVal(c)                        # D = C^2 (mag: 1)
-        e.mul2(x1, b)                         # E = X1*B (mag: 1)
-        negE.set(e).negate(1)                 # negE = -E (mag: 2)
-        f.mul2(x2, b)                         # F = X2*B (mag: 1)
-        x3.add2(e, f).negate(3).add(d)      # X3 = D-E-F (mag: 5)
-        negX3.set(x3).negate(5).normalize()    # negX3 = -X3 (mag: 1)
-        y3.set(y1).mul(f.add(negE)).negate(3) # Y3 = -(Y1*(F-E)) (mag: 4)
-        y3.add(e.add(negX3).mul(c))          # Y3 = C*(E-X3)+Y3 (mag: 5)
-        z3.mul2(z1, a)                        # Z3 = Z1*A (mag: 1)
+        # fmt: off
+        negX1.set(x1).negate(1)                 # negX1 = -X1 (mag: 2)
```

Comment:
That sucks that you can't preserve comment alignment. I'm tempted to just give in to our new overlords here to avoid the `# fmt: off` tags, but this is fine too.

Reply:
Yes, either works.

Reply:
Let's just leave it the way you have it. I'm just trying to be in the habit of avoiding these tags, but you don't seem to be abusing them.